google_monitoring/README.md

```diff
@@ -1,3 +1,4 @@
+<!-- BEGIN_TF_DOCS -->
```

Comment:
piping to the readme like `$ terraform-docs markdown . |tee README.md` will not add tags to the `README.md` file :slightly_smiling_face: